src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -93,43 +93,40 @@ <h2 class="h3">Active courses</h2>
                 </td>
                 <td>
                   <span *ngIf="!courseStats[course.course.courseId]">
-                    <a *ngIf="!course.isLoadingCourseStats; else loadingSpinner" (click)="getCourseStats(i)"
-                      [tmRouterLink]="">
-                      Show
-                    </a>
+                    <button *ngIf="!course.isLoadingCourseStats; else loadingSpinner"
+                      (click)="getCourseStats(i)" class="link-button" > Show
+                    </button>
                   </span>
                   <span *ngIf="courseStats[course.course.courseId]">
                     {{ courseStats[course.course.courseId]['sections'] }}
                   </span>
                 </td>
                 <td>
                   <span *ngIf="!courseStats[course.course.courseId]">
-                    <a *ngIf="!course.isLoadingCourseStats; else loadingSpinner" id="show-statistics-{{ i }}"
-                      (click)="getCourseStats(i)" [tmRouterLink]="">
-                      Show
-                    </a>
+                    <button *ngIf="!course.isLoadingCourseStats; else loadingSpinner"
+                      (click)="getCourseStats(i)" class="link-button" > Show
+                    </button>
                   </span>
                   <span *ngIf="courseStats[course.course.courseId]">
                     {{ courseStats[course.course.courseId]['teams'] }}
                   </span>
                 </td>
                 <td>
                   <span *ngIf="!courseStats[course.course.courseId]">
-                    <a *ngIf="!course.isLoadingCourseStats; else loadingSpinner" (click)="getCourseStats(i)"
-                      [tmRouterLink]="">
-                      Show
-                    </a>
+                    <button *ngIf="!course.isLoadingCourseStats; else loadingSpinner"
+                      id="show-statistics-{{ i }}" (click)="getCourseStats(i)"
```

Comment:
Can I check if this `id` attribute should be here or for the previous column? I think it was previously on the "Teams" column... doubt it makes much of a difference though, but just in case, should we shift it back?

Reply:
Updated